daemon/config.go

```diff
@@ -33,7 +33,16 @@ type CommonConfig struct {
 	Root           string
 	TrustKeyPath   string
 	DefaultNetwork string
-	NetworkKVStore string
+
+	// ClusterStore is the storage backend used for the cluster information. It is used by both
+	// multihost networking (to store networks and endpoints information) and by the node discovery
+	// mechanism.
+	ClusterStore string
```

Comment:
Typically, the convention in Go is to identify stringly-typed network endpoint fields with `Addr`. This should be `ClusterStoreAddr`.